bin/katello-installer

```diff
@@ -5,6 +5,10 @@ require 'kafo'
 
 CONFIG_FILE = './config/katello-installer.yaml'
 
+# Add --profile option on EL6 due to segfaulting
+os = `uname -a`
```

Comment:
It doesn't happen so much with RHEL/CentOS but if a user compiles a custom kernel, this check might not catch it.

What do you think about looking at /etc/redhat-release instead and make sure it =~ `/ 6\./` or similar?